.github/workflows/docs.yml

```diff
@@ -0,0 +1,32 @@
+name: Docs
+on:
+  push:
+    branches:
+      - main
+      - issue-64
```

Comment:
should probably remove this? only want to build docs on the main branch no?

Reply:
Yep it needs to go 👍